src/awx_plugins/inventory/plugins.py

```diff
@@ -18,6 +18,8 @@ class PluginFileInjector:
     plugin_name = None  # Ansible core name used to reference plugin
     # base injector should be one of None, "managed", or "template"
     # this dictates which logic to borrow from playbook injectors
+    plugin_description = None
+    # this field is for the description of the injector
```

Comment:
In general, comments should be on the same line if possible:
```suggestion
    plugin_description = None  # this field holds the injector description
```
This is because over time, the comment line and the code line may drift away from each other by accident. And the comments will end up being in weird places.